app/models/setting.rb

```diff
@@ -139,24 +122,20 @@ def update_all
       settings = load_settings_hash
       # per site settings
       Site.where(active: true).each do |site|
-        if get_hash_of_settings_in_db(site.id) != get_hash_of_settings_in_yaml(settings)
-          Rails.logger.info("Reloading settings for site #{site.id}...")
-          update_site_from_hash(site, settings)
-        end
+        Rails.logger.info("Reloading settings for site #{site.id}...")
+        update_site_from_hash(site, settings)
       end
       # globals
-      if get_hash_of_settings_in_db != get_hash_of_settings_in_yaml(settings, true)
-        Rails.logger.info("Reloading global settings...")
-        global_settings_in_db = Setting.where(global: true).to_a
-        each_setting_from_hash(settings, true) do |section_name, setting_name, setting|
-          unless global_settings_in_db.detect { |s| s.section == section_name and s.name == setting_name }
-            global_settings_in_db << Setting.create!(setting.merge(section: section_name, name: setting_name))
-          end
+      Rails.logger.info("Reloading global settings...")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.